### ISSUE TITLE
Hide train scroll elements when empt

### ISSUE THREAD

**Issue by ST-DDT** [Zhincore/dynmap-trains#6]
Is it possible to hide the train scroll elements when the list is empty/`length <= 1`/has barely any elements?

![grafik](https://github.com/Zhincore/dynmap-trains/assets/1579362/b469bcb2-4ce6-4a43-8d9e-d78386970915)

**Comment by Zhincore** [comment]
Good point, I'll take a look at it at some point, but if I recall correctly I had some trouble with this
